src/libraries/System.Private.CoreLib/src/System/AppDomain.cs

```diff
@@ -424,11 +419,7 @@ public void SetThreadPrincipal(IPrincipal principal)
                         if (s_getWindowsPrincipal == null)
                         {
                             Type type = Type.GetType("System.Security.Principal.WindowsPrincipal, System.Security.Principal.Windows", throwOnError: true)!;
-                            MethodInfo? mi = type.GetMethod("GetDefaultInstance", BindingFlags.NonPublic | BindingFlags.Static);
-                            if (mi == null)
-                            {
-                                throw new PlatformNotSupportedException(SR.PlatformNotSupported_Principal);
-                            }
+                            MethodInfo mi = type.GetMethod("GetDefaultInstance", BindingFlags.NonPublic | BindingFlags.Static) ?? throw new PlatformNotSupportedException(SR.PlatformNotSupported_Principal);
```

Comment:
What's the line length limit that we think is ok? This line is 178 characters. It won't fit on my laptop screen... . 

(There are other places in this delta that are introducing lines that are too long to my taste, but this one seems to be pushing it.)

Reply:
This is what the code fixer did, but I can go through and change them to put what comes after the ?? on its own line if that feels better.

Reply:
I think a reasonable line length is <120. Over 150 is too long.